modules/swagger-codegen/src/main/resources/typescript-angular/package.mustache

```diff
@@ -40,9 +40,9 @@
     "@angular/platform-browser": "^{{ngVersion}}",{{#useNgPackagr}}
     "ng-packagr": {{#useOldNgPackagr}}"^1.6.0"{{/useOldNgPackagr}}{{^useOldNgPackagr}}"^2.4.1"{{/useOldNgPackagr}},{{/useNgPackagr}}
     "reflect-metadata": "^0.1.3",
-    "rxjs": "^5.4.0",
+    "rxjs": "{{#useRxJS6}}^6.1.0{{/useRxJS6}}{{^useRxJS6}}^5.4.0{{/useRxJS6}}",
     "zone.js": "^0.7.6",
-    "typescript": "^2.1.5"
+    "typescript": "~2.7.2"
```

Comment:
since we are requiring angular cli, constraining to `"typescript": "^2.1.5"` should be fine and more flexible, especially when choosing between angular 5 or 6

Reply:
The problem is, with ^2.1.5, typescript 2.8.3 will be installed during the tests which is not compatible to angular core. I'll change it to >= 2.1.5 < 2.8, would that be more appropriate?

Reply:
yes, this sounds better